src/app/containers/Header/index.test.jsx

```diff
@@ -166,5 +169,68 @@ describe(`Header`, () => {
       const skipLink = document.querySelector("a[href$='#content']");
       expect(skipLink).toHaveAttribute('lang', 'ru-UA');
     });
+
+    it('should focus on consent banner heading on mount', () => {
+      const initialFocusElement = document.activeElement;
+      const { getByText } = render(
+        HeaderContainerWithContext({
+          pageType: INDEX_PAGE,
+          service: 'pidgin',
+          serviceContext: pidginServiceConfig,
+        }),
+      );
+      const pidginPrivacyHeading =
+        pidginServiceConfig.default.translations.consentBanner.privacy.title;
+      expect(document.activeElement).toBe(getByText(pidginPrivacyHeading));
+      expect(document.activeElement).not.toBe(initialFocusElement);
```

Comment:
Is this assertion necessary? If the first assertion fails, this second one can never be reached anyway?

Reply:
This assertion is to ensure that the focus has been pulled and is not on the heading by default, to check that the focus management is actively doing something, I can switch the order of the assertions though 👍

Reply:
Imo if you asserting something is one value there's not a great deal of value of asserting it isn't something else? I think by default focus would be on the body of document the test runs in, I'm not sure you need to explicitly test that behaviour; in other words I think focus can only be pulled to the header by your code not any implicit behaviour.